infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/LocationTrackGeometry.kt

```diff
@@ -726,6 +728,9 @@ sealed class LayoutNode {
     fun containsInnerJoint(switchId: IntId<LayoutSwitch>, joint: JointNumber): Boolean =
         portA.let { port -> (port as? SwitchLink)?.matches(switchId, joint) ?: false }
 
+    fun containsOuterJoint(switchId: IntId<LayoutSwitch>, joint: JointNumber): Boolean =
```

Comment:
Käytetäänkös tätä jossain? Outer ja inner (koskee siis myös noita ylempiä) on vähän jänniä käsitteitä LayoutNoden kannalta koska noodilla on vain 2 porttia joihin kiinnittyä ja riippuu edgen kiinnityssuunnasta kumpi on in ja kumpi out.